modules/programs/vim.nix

```
@@ -79,6 +85,26 @@ in {
         '';
       };
 
+      optionalPlugins = mkOption {
+        type = with types; listOf (package);
+        default = defaultOptionalPlugins;
```

```suggestion
        default = [ ];
```

I think we can just set a default value right here.